docs/management/sentinel.md

```diff
@@ -1237,4 +1237,4 @@ API that many kernels offer. However it is not still clear if this is a good
 solution since the current system avoids issues in case the process is just
 suspended or not executed by the scheduler for a long time.
 
-**A note about the word slave used in this man page**: Starting with Redis 5, if not for backward compatibility, the Redis project no longer uses the word slave. Unfortunately in this command the word slave is part of the protocol, so we'll be able to remove such occurrences only when this API will be naturally deprecated.
+**A note about the word slave used in this man page**: PLEASE-REVIEW{Starting with Redis 5, if not for backward compatibility, the Redis project no longer uses the word slave. Unfortunately in this command the word slave is part of the protocol, so we'll be able to remove such occurrences only when this API will be naturally deprecated.}
```

Comment:
This note is in many doc pages. I think we should replace all of them in a separate PR.

Reply:
So only a change to Redis OSS 5 in this PR.

If we are going to do all of them in a single PR, I would suggest having all such pages either link to an explanation or include one from a single file.  My preference would be a link to a page that also explains the movement from MASTER to PRIMARY.

Either a link or include would also let us improve the message in just one place.  It seems to me that "... when this API will be naturally deprecated." should be changed to "... MASTER and SLAVE have been deprecated in favor of PRIMARY and REPLICA and will be deleted when the community is ready to accept such a breaking change."